screen-home/src/main/java/com/ivy/home/HomeHeader.kt

```diff
@@ -157,6 +157,17 @@ private fun HeaderStickyRow(
 
             // Balance mini row
             if (percentExpanded < 1f) {
+                val overflowLengthOfBalance = 7
+                val overflowLengthOfMonthRange = 12
+                val lengthOfCurrencyAndBalance = (currency + balance.toString()).length
```

Comment:
Use `remember` + `deriveStateOf` to make it more optimal.
```kotlin
val overflow by remember(... things that depend ...) {
   deriveStateOf {
        do calculations here
    }
}
```

Reply:
i am not able to resolve these lint issues

Reply:
@Raghav1783 Detekt is pretty easy. Just extra magic numbers as constants and remove unused imports.
```
home/runner/work/ivy-wallet/ivy-wallet/screen-budgets/src/main/java/com/ivy/budgets/BudgetModal.kt:1:1: Unused import [NoUnusedImports]
/home/runner/work/ivy-wallet/ivy-wallet/screen-budgets/src/main/java/com/ivy/budgets/BudgetModal.kt:1:1: Unused import [NoUnusedImports]
/home/runner/work/ivy-wallet/ivy-wallet/screen-home/src/main/java/com/ivy/home/HomeHeader.kt:162:47: This expression contains a magic number. Consider defining it to a well named constant. [MagicNumber]
/home/runner/work/ivy-wallet/ivy-wallet/screen-home/src/main/java/com/ivy/home/HomeHeader.kt:163:50: This expression contains a magic number. Consider defining it to a well named constant. [MagicNumber]
```

Lint, I don't know screenshot me the Lint report. If it's hard to resolve you add a baseline to suppress them.